WalletWasabi.Fluent/Helpers/AuthorizationHelpers.cs

```diff
@@ -16,7 +16,7 @@ public static AuthorizationDialogBase GetAuthorizationDialog(IWalletModel wallet
 		}
 		else
 		{
-			return new PasswordAuthDialogViewModel(wallet, PasswordRequestIntent.Send);
+			return new PasswordAuthDialogViewModel(wallet, "SEND");
```

Comment:
Do not capitalize.
- https://github.com/zkSNACKs/WalletWasabi/pull/11778#discussion_r1368789128
- https://github.com/zkSNACKs/WalletWasabi/pull/11778#pullrequestreview-1694228865
- https://github.com/zkSNACKs/WalletWasabi/pull/11594#discussion_r1368393686